kornia/augmentation/augmentation.py

```diff
@@ -21,10 +21,11 @@
 
 
 class AugmentationBase(nn.Module):
-    def __init__(self, apply_fcn: Callable, return_transform: bool = False) -> None:
```

Comment:
I think we might put the ```return_transform``` in ```forward``` than an init parameter, it would make more sense that the user could use it as a switch for each run.

Currently, we can use it like:
```python
nn.Sequential(
    lambda x: kornia.RandomAffine(x, return_transform=True),
    lambda x: kornia.ColorJitter(x, return_transform=True)
)
```

Or we can implement our own sequential:
```python
kornia.Sequential([
    kornia.RandomAffine(x),
    kornia.ColorJitter(x)
], return_transform=True)
```